.github/workflows/web.yaml

```diff
@@ -0,0 +1,53 @@
+# Separate workflow for the wasmer-web crate.
+#
+
+# Automatically cancel previous workflow runs when a new commit is pushed.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+name: Wasmer Web
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+jobs:
+  web:
+    name: Build and Test (wasmer-web)
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        rust: [nightly]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Configure Git clone credentials
+      uses: de-vri-es/setup-git-credentials@v2
+      with:
+        credentials: ${{secrets.GIT_CREDENTIALS}}
```

Comment:
This is making the web test fail. We can remove it as I don't think is necessary

Reply:
done